django_project_base/account/rest/project_profiles_utils.py

```diff
@@ -59,7 +58,7 @@ def get_project_members(request: Request) -> QuerySet:
 
     if getattr(request, "current_project_slug", None):
         # if current project was parsed from request, filter profiles to current project only
-        qs = qs.filter(projects__project__slug=request.current_project_slug)
+        qs = qs.filter(projects__project=project)
```

Comment:
Tole je potencialno hitrejše in potencialno bolj napačno, kot prejšnja rešitev. Predvidevam, da ta sprememba ni bila nujna. Razen če ti je odletavalo, ker v back-end ni prišel currently selected project pravilno.